cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -1,4 +1,4 @@
-replicas: 1
+replicas: 2
```

Comment:
I feel like we can probably leave this at 1 replica by default - but maybe make this configurable? The nature of Crossplane is such that we don't really need to fail over instantly to a second replica; we can usually tolerate being offline for a few seconds while a replacement spins up.

Reply:
I had a similar idea and actually wanted to add a new variable too `rbacManager.replicas` to effectively distinguish the two.